docs/README.md

```diff
@@ -1,58 +1,44 @@
-# VTEX React App Template
+# VTEX Product List Context app
 
-Our guide repository to structure for react apps, that should be used as a template.
+This app is the Product List Context that should be used to store certain products. Example of usage: This is used on the `Shelf` app to store the visible products.
 
-We use `yarn` as our default package manager, before coding make sure to run yarn on: `root` and `react` folders.
+## API
 
-## Some features:
+`ProductListProvider`: Context provider that should be used to wrap your components that will use the hooks below.
```

Comment:
This Hooker and context info I would add in a section called ## Modus operandi. 
This ## Modus operandi section allows users to fully understand the **practical application** of the block/app in their store.
If you feel compelled to give further details about the app, such as how the hooker and product list provider works, I believe you should use this section.